### DISCUSSION TITLE
Renaming a component

### DISCUSSION THREAD

**Discussion by tassoman** [quasarframework/quasar#14291] [General - Components / Directives / etc] [answered]
Hello there, I've just renamed a component name, its name inside export, and inside `components: {}` but still can't be found. 
Is there some components registry to be flushed inside Q 2.7.7 
I've also tried with `yarn quasar clean` without solution, I have a console warning and the component doesn't show up in the components tree of browser's inspection tool. 
Previous name was _CheckLinkItem_ with capital L. 
I have read and write privileges in the project. 

>  App •  WARNING  •  UI  in ./src/components/ReportList.vue?vue&type=script&lang=js
>
> export 'ChecklinkItem' (imported as 'ChecklinkItem') was not found in 'src/components/ChecklinkItem' (possible exports: default)

**Comment by MilosPaunovic** [comment]
Share all the imports at least; there should not be a problem with renaming so any extra piece of information might help debugging process.

**Reply by tassoman** [reply]
Sorry, I can't get what you mean with "share all the imports", inside parent component I've already imported it
```javascript
import { useReportStore } from "src/stores/report-store";
import { api } from "src/boot/axios";
import { ChecklinkItem } from "src/components/ChecklinkItem";
```

**Reply by tassoman** [reply]
```bash
tassoman@hostname:~/devel/GitLab/myapp/frontend/src/components$ ls -lah
total 28K
drwxr-xr-x 2 tassoman tassoman 4.0K Aug 30 10:44 .
drwxr-xr-x 9 tassoman tassoman 4.0K Aug 22 14:21 ..
-rw-r--r-- 1 tassoman tassoman  653 Aug 30 10:47 ChecklinkItem.vue
-rw-r--r-- 1 tassoman tassoman  742 Aug 22 14:21 EssentialLink.vue
-rw-r--r-- 1 tassoman tassoman 1.7K Aug 22 14:21 LinkList.vue
-rw-r--r-- 1 tassoman tassoman 1.6K Aug 30 10:49 ReportList.vue
-rw-r--r-- 1 tassoman tassoman 2.4K Aug 26 13:23 UrlForm.vue
```

**Comment by MilosPaunovic** [comment, marked as answer]
Seems like this would do the trick:

`import ChecklinkItem from "src/components/ChecklinkItem"`

**Reply by tassoman** [reply]
whoops bad typo! Thanks! 💡
